cli/tests/pcluster/aws/test_fsx.py

```diff
@@ -124,3 +124,38 @@ def test_describe_volumes(boto3_stubber):
     # Fourth boto3 call after resetting the AWSApi instance. The latest fsx lifecycle should be retrieved from boto3
     AWSApi.reset()
     assert_that(AWSApi.instance().fsx.describe_volumes([volume])[0]["Lifecycle"]).is_equal_to("CREATED")
+
+
+def get_describe_file_caches_mocked_request(file_cache_ids, lifecycle):
```

Comment:
Why do we need the lifecyle in the tests?
I'm asking because I do not expect the CLI to do any check on the status of the FC, but only to verify that it exists and that it can be descirbed.

Reply:
I assumed while checking if the cache exists we will also check for the Lifecycle of the Cache like it should not be in Failed or Deleting phase while we are creating the cluster. If this is not required I can remove the Lifecycle check in these tests.